paimon-core/src/main/java/org/apache/paimon/schema/SchemaValidation.java

```diff
@@ -100,6 +101,9 @@ public static void validateTableSchema(TableSchema schema) {
             }
         }
 
+        // Valid the format type
+        checkNotNull(options.formatType());
```

Comment:
option with default value should never return null.
we don't need this checking.